crates/relayer-cli/src/commands/query/packet/acks.rs

```diff
@@ -72,8 +65,11 @@ impl QueryPacketAcknowledgementsCmd {
 // cargo run --bin hermes -- query packet acknowledgements --chain ibc-0 --port transfer --connection ibconexfer --height 3
 impl Runnable for QueryPacketAcknowledgementsCmd {
     fn run(&self) {
+        use crate::conclude::json;
```

Comment:
Should we move this at the top, with the other `use` calls?

Reply:
It's only used in this function, so I would keep it here. Local imports for one-time use are fine imho.